src/main/java/com/getindata/connectors/http/internal/utils/JavaNetHttpClientFactory.java

```diff
@@ -45,6 +47,13 @@ public static HttpClient createClient(HttpLookupConfig options) {
                 .getOptional(HttpLookupConnectorOptions.SOURCE_LOOKUP_CONNECTION_TIMEOUT)
                 .ifPresent(clientBuilder::connectTimeout);
 
+        options.getReadableConfig()
+                .getOptional(HttpLookupConnectorOptions.SOURCE_LOOKUP_PROXY)
+                .ifPresent(proxy -> {
```

Comment:
Thanks a lot for your contribution!
For completeness, I believe password-protected proxies may also be supported. What do you think? How much effort will be needed?